src/Core/Services/RestService.cs

```diff
@@ -395,6 +395,20 @@ public bool TryGetRestRouteFromConfig([NotNullWhen(true)] out string? configured
             return false;
         }
 
+        /// <summary>
+        /// Helper method to extract the configured base route
+        /// </summary>
+        public string GetBaseRouteFromConfig()
+        {
+            if (_runtimeConfigProvider.TryGetConfig(out RuntimeConfig? config)
+                && config.Runtime.BaseRoute is not null)
```

Comment:
Are we not checking for config.Runtime is not null because it can never be?

Reply:
Yeah, since `Runtime` is a non-nullable property of `RuntimeConfig`, that check is skipped.